test/puppeteer/docs.test.ts

```diff
@@ -1,11 +1,63 @@
 // Imports
 import path from 'path';
 
+// Destructured constant for readability
+const { keyboard } = page;
+
+// Helper functions used in multiple tests
+const menuDisplayState = () => page.evaluate(() => document.querySelector<HTMLDivElement>('#menu').style.display);
+const currentFocusId = () => page.evaluate(() => document.activeElement.id);
```

Comment:
Acronyms and initialisms that occur after the first word in a variable name should be all caps.

```suggestion
const currentFocusID = () => page.evaluate(() => document.activeElement.id);
```

This will require that you update this name elsewhere in this file, too.